src/modules/form/react-select/ts/index.tsx

```diff
@@ -36,24 +36,40 @@ function ReactSelect(props) {
 	}, []);
 
 	let value = props.options.find(item => item.value === props.value);
-	const onChange = ({ label, value }) => {
+	const onChange = params => {
```

Comment:
Hi @veD-tnayrB 

I've noticed we can optimize our `onChange` handler to be more concise. To avoid the repetition of the `props.onChange` method call, I suggest using a ternary operator to determine the values at the start of the method. Furthermore, destructuring `props.name` once and crafting a single target object for reuse in `currentTarget` would make the code cleaner and avoid redundancy.

Would you consider implementing these adjustments? This should lead to easier maintenance and improved readability.

Kind Regards,


---

Reply:
Hi @jircdev 

The changes to both the `onChange` and the `use-styles hook` have been made and are already uploaded:

I put the use-styles in a separated file that u can find at: 
`src\modules\form\react-select\ts\use-styles.ts`.

And on the other hand this is the final result of the adjustments in the onChange based on the description provided: 

```ts
const onChange = params => {
		if (!props.onChange) return;
		const isMultiValues = properties?.isMulti;
		const values = isMultiValues && params.map(selectedItem => selectedItem.value);
		const value = isMultiValues ? values : params.value;
		const { name } = props;

		const target = {
			value,
			name,
		};

		props.onChange({
			target,
			currentTarget: target,
		});
	};
```

Kind Regards.